src/components/Header/Header.module.scss

```diff
@@ -0,0 +1,19 @@
+@import url('https://fonts.googleapis.com/css2?family=KoHo&display=swap');
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding: 0 1rem;
+  height: 5rem;
+  width: 100vw;
```

Comment:
Be careful. I think that 100vw does not scale when you make the window larger. Maybe only in some specific scenarios. But if there is no specific reason for 100vw, use %.